### PR TITLE
SECURITY.md Revision 1

### PR DESCRIPTION
**Pull request by quantum9Innovation**
This is the first major revision of our SECURITY.md document, and it's designed to meet our organization-wide commitment to better security. You can view the new document [here](https://github.com/epispot/epispot/blob/2adbdacaa19c43dc35ed40dbd8cbd1d3cb061e40/SECURITY.md).

## Changelist:
- SECURITY.md
    - Include more detailed instructions on the vulnerability disclosure process
    - Remove issue creation as a way to expose a vulnerability
    - Add emails and GPG keys to our vulnerability submission options
- Remove 'known vulnerabilities' badge (cannot be tracked anymore) from README-nightly.md
- setup.py, setup.cfg, setup-nightly.py
    - Add `author_email`
- Make @Quantalabs a co-owner of the security policy on .github/CODEOWNERS

## Additional Notes

This PR adds @Quantalabs as a code owner to SECURITY.md, and as such @Quantalabs will be responsible for the review process.

### CODE REVIEW

**Comment by quantum9Innovation** [comment]
@Quantalabs waiting on your review

File: SECURITY.md [hunk 1]
```diff
@@ -17,8 +17,92 @@ The following versions will receive regular security updates:
...
+
+Please note that we will rarely reply to your email using a GPG key unless it poses a threat to other users and/or packages. If you feel that it is necessary to use a GPG key or want to conceal confidential information, our key fingerprint is:
+
+```text
+sec   rsa3072/25B6A3ACE2E9FA1C 2021-07-30 [SC]
+      6B3C84252163F3BF00AB8CC125B6A3ACE2E9FA1C
+uid                 [ultimate] epispot (This is epispot's official GPG key for security vulnerability submissions.) 
+<dev.quantum9innovation@gmail.com>
+ssb   rsa3072/C85634FF0F7A1E8A 2021-07-30 [E]
+```
```
Comment (Quantalabs) [inline SECURITY.md:44]:
```suggestion
```
This is useless

File: setup-nightly.py [hunk 2]
```diff
@@ -1,34 +1,35 @@
 import setuptools
 
-with open("README-nightly.md", "r") as fh:
+with open('README-nightly.md', 'r') as fh:
```
Comment (Quantalabs) [inline setup-nightly.py:3]: How is this related?

**Review by Quantalabs — CHANGES_REQUESTED** [review]

File: SECURITY.md (hunk 1 above)
Comment (quantum9Innovation) [reply SECURITY.md:44]: Ok fix pending

File: setup-nightly.py (hunk 2 above)
Comment (quantum9Innovation) [reply setup-nightly.py:3]: As a side effect to adding the `author` label on our manifest files, we've also replaced double quotes with single ones

File: SECURITY.md [hunk 3]
```diff
@@ -17,8 +17,82 @@ The following versions will receive regular security updates:
...
+
+**Do not** create an issue, discussion, or any other public notice on the GitHub repository. Please also do not mention the vulnerability publicly in any other projects or channels (that includes official epispot projects as well).
+
+#### I don't have a GitHub account
+
+The simplest way to report a vulnerability is to email the [epispot package authors](https://pypi.org/project/epispot) directly with your concern. The current epispot package author is @quantum9innovation. Send your emails to dev.quantum9innovation@gmail.com and we'll respond as soon as possible. Please include `security`[*](#additional-notes) in the subject line or body of your email so that we can quickly identify the issue.
+
+#### I want to use a GPG key
+
+**Note:** Please contact us in a separate email for the passphrase to the key shown below.
```
Comment (Quantalabs) [inline SECURITY.md:36]: Also no need for this

**Review by Quantalabs — CHANGES_REQUESTED** [review]

File: SECURITY.md (hunk 3 above)
Comment (Quantalabs) [reply SECURITY.md:36]:
```suggestion
```

**Review by Quantalabs — APPROVED** [review]

**Review by Quantalabs — APPROVED** [review]
lgtm :shipit:
